root/layout/components/sidebar/CDStubSidebar.js

```diff
@@ -35,11 +36,19 @@ const CDStubSidebar = ({cdstub}: Props) => {
     ended: true,
   }) : null;
 
+  const artistField =
+    escapeLuceneValue(cdstub.artist || l('Various Artists'));
+  const releaseField = escapeLuceneValue(cdstub.title);
+  const tracksMediumField = escapeLuceneValue(cdstub.track_count);
+  const barcodeField = cdstub.barcode
+    ? escapeLuceneValue(cdstub.barcode)
+    : null;
+
   const searchQuery = (
-    'artist:(' + escapeLuceneValue(cdstub.artist || l('Various Artists')) + ') ' +
-    'release:(' + escapeLuceneValue(cdstub.title) + ') ' +
-    'tracksmedium:(' + escapeLuceneValue(cdstub.track_count) + ')' +
-    (cdstub.barcode ? ' barcode:(' + escapeLuceneValue(cdstub.barcode) + ')' : '')
+    `artist:(${artistField}) ` +
+    `release:(${releaseField} ` +
```

Comment:
Closing parenthesis went missing since commit https://github.com/metabrainz/musicbrainz-server/pull/1248/commits/2170478f2d73555571f7140acb0e5b34301f7e74
```suggestion
    `release:(${releaseField}) ` +
```

Reply:
Good catch, fixed